handlers/kafka_helper.go

```diff
@@ -22,6 +22,15 @@ const ProducerSchemaName = "payment-processed"
 // paymentProcessed represents the avro schema which can be found in the chs-kafka-schemas repo
 type paymentProcessed struct {
 	PaymentSessionID string `avro:"payment_resource_id"`
+	RefundId         string `avro:"refund_id,omitempty"`
+}
+
+func producePaymentMessage(paymentID string) error {
+	return produceKafkaMessage(paymentID, "")
```

Comment:
I thought we didn;t have to add an empty string as the refundId on the Payment message? I thought we could ignore it from the Payments API no bother, but it was get serialised to an empty string when consuming the kafka message?

Reply:
Would you see it as a separate method then? There are no optional variables in Go.

Reply:
ah if it's going to require a new method then forget about it.